hubble-relay: deprecate peer svc through local unix domain socket (#23407)
cilium-team/cilium: Documentation/internals/hubble.rst

scenario).

Hubble Relay was first introduced as a technology preview with the release of

> Cilium v1.8 and has been declared stable with the release of Cilium v1.9.

Reviewer: Nit:
```suggestion
Cilium v1.8 and was declared stable with the release of Cilium v1.9.
```
(It happened at a definite time in the past.)

But this is not blocking, let me know when you want this merged.

Author: Thanks @qmonnet updated
